main.py

```diff
@@ -144,7 +146,13 @@ async def get_task_exceptions(task_id: str) -> List[str]:
     return tasks[task_id]
 
 
-@app.get("/compiled_artifact/{task_id}", response_class=JSONResponse)
+class ManifestResponse(JSONResponse):
+    pass
+
+
+# NOTE: `response_model=None` so that we only use our own validation
+#   from ethpm_types.
+@app.get("/compiled_artifact/{task_id}", response_model=None)
```

Comment:
this is the fix right here. we dont need response to be another model because we are validating the manifest completely in our own code via the task. so this is even more performant!